bigtable/google/cloud/bigtable/instance.py

```diff
@@ -276,6 +282,14 @@ def create(
         """
 
         if clusters is None:
+            warnings.warn(
+                _INSTANCE_CREATE_WARNING.format(
+                    "location_id", "serve_nodes", "default_storage_type"
+                ),
+                PendingDeprecationWarning,
```

Comment:
I think we can make this just a regular `DeprecationWarning`.  Thanks for adding `stacklevel=2`!

Reply:
Done, thanks